feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x60**.
+    *   Inner header ToS is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **20**.
+    *   Inner header TTL is **9**.
+
+### PF-1.3.8: IPv6 traffic GUE encapsulation with explicit ToS and TTL configuration on tunnel
+
+DUT and ATE actions:
+
+*   Re-configure the IPv4 GUE tunnel on the DUT with ToS value *0x60*.
+*   Re-configure the IPv4 GUE tunnel on the DUT with TTL value of *20*.
+*   Generate **IPv6 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x60**.
+    *   Inner header Traffic Class is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **20**.
+    *   Inner header Hop Limit is **9**.
+
+### PF-1.3.9: IPv4 traffic that should be GUE encapsulation but TTL=1
+
+ATE action:
+
+*   Generate **IPv4 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using source address of ATE:Port1 at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *1*.
+
+Verify:
+
+*   All packets should have TTL decremented to 0, dropped and ICMP Time
+    Exceeded (Type 11) / Time to Live exceeded in Transit (Code 0) sent back to
+    source address (ATE:Port1).
+
+### PF-1.3.10: IPv6 traffic that should be GUE encapsulation but Hop Limit=1
+
+ATE action:
+
+*   Generate **IPv6 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using source address of ATE:Port1 at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set Hop Limit of the packets to *1*.
+
+Verify:
+
+*   All packets should have Hop Limit decremented to 0, dropped and
+    ICMPv6 Time Exceeded (Type 3) / hop limit exceeded in transit (Code 0) sent
+    back to source address (ATE:Port1).
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+    # TODO propose new OC paths for GUE encap based on the protocol next hop of a route
```

Comment:
It is true, new OC paths will be needed.  FYI, they should [follow the pattern being established ](https://github.com/openconfig/public/pull/1153) in the /afts tree, but using /policy-forwarding instead.

Reply:
Ack, thanks for the pointer. I'll follow this to send proposal for the OC paths.